io.github.lainsce.Emulsion.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "io.github.lainsce.Emulsion",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Emulsion",
+    "rename-icon" : "io.github.lainsce.Emulsion",
+    "finish-args" : [
+        "--filesystem=xdg-download",
```

Comment:
what does it use xdg-download for?

Reply:
That's for a future feature, exporting palettes.

Reply:
can be removed until needed, exporting palettes can use the native file chooser anyway

Reply:
Alright